modules/quality/test/test_brisque.cpp

```diff
@@ -77,6 +77,18 @@ TEST(TEST_CASE_NAME, compute_features)
     EXPECT_EQ(features.cols, 36);
 }
 
+// check empty method
+TEST(TEST_CASE_NAME, check_empty)
+{
+    auto ptr1 = create_brisque();
+    EXPECT_FALSE(ptr1->empty());
+
+    const auto model = cvtest::findDataFile(MODEL_FNAME, false);
+    const auto range = cvtest::findDataFile("some_wrong_range_name", false);
```

Comment:
Don't use `findDataFile` here, it will throw a `SkipTestException` and the code below this line won't run.  See https://github.com/opencv/opencv/blob/08a3eef62a378c66df5d4e7a786b57d1c68819a5/modules/ts/src/ts.cpp#L1053